src/Application/Common/Exceptions/ValidationException.cs

```diff
@@ -1,14 +1,20 @@
-﻿using System.Text.Json;
+﻿using CleanCompanyName.DDDMicroservice.Domain.Common.Validators;
 
 namespace CleanCompanyName.DDDMicroservice.Application.Common.Exceptions;
 
 public class ValidationException : Exception
 {
-    public List<ValidationError> Errors { get; }
+    public ValidationException()
+        : base("Some problems has been found during validation process:")
+    {
+        Errors = new List<ValidationError>();
+    }
 
-    public ValidationException(List<ValidationError> errors)
-        : base("Some problems has been found during validation process. Errors: " + JsonSerializer.Serialize(errors))
+    public ValidationException(IEnumerable<ValidationError> errors)
+        : this()
```

Comment:
```suggestion
```

Reply:
The idea of having this is to have some "title" message and the errors list for our log message. Any kind of other suggestion to reach it?

Reply:
I went this way because actually, we are only using this kind of Exception when we are receiving ValidationErrors, so we should pass the errors. Now there is only one constructor.
`  public ValidationException(IEnumerable<ValidationError> errors)
        : base("Some problems has been found during validation process:")
    {
        Errors = errors;
    }`